### PR TITLE
echo: add exit code command

### PR DESCRIPTION
**Pull request by shreenidhi11**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by shreenidhi11** [comment]
Hi, This is my first time contributing to the page. I have worked with Linux for nearly 3 years. I wanted to share one of my learning of the echo command. The echo command when given $? as a parameter, prints success or failure message of a recently executed command:

For example:
shreenidhi@Shreenidhis-MacBook-Pro tldr % echo "Hi"
Hi
shreenidhi@Shreenidhis-MacBook-Pro tldr % echo $?
0
Here the recent command is echo "Hi" and the result of executing echo $? means that it's a successfully executed command.

File: pages/common/echo.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Enable interpretation of backslash escapes (special characters):
 
 `echo -e "{{Column 1\tColumn 2}}"`
+
+- Print success or failure message of a recently executed command:
```
Comment (blueskyson) [inline pages/common/echo.md:26]:
```suggestion
- Print success or failure message of the last executed command:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

**Comment by acuteenvy** [comment]
This won't work on all shells. For example, in Windows `cmd` this variable is called `%errorlevel%`, not `$?`.

**Comment by kbdharun** [comment]
> This won't work on all shells. For example, in Windows `cmd` this variable is called `%errorlevel%`, not `$?`.

In Windows, it does work in Powershell and in Git Bash (See Ref). But not in Command Prompt (where it is `echo %errorlevel%%` as you mentioned). Maybe we can highlight this detail inside brackets `()`.

```powershell
PS C:\Users\kbdha> echo hi
hi
PS C:\Users\kbdha> echo $?
True
```

File: pages/common/echo.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - Enable interpretation of backslash escapes (special characters):
 
 `echo -e "{{Column 1\tColumn 2}}"`
+
+- Print success or failure message of the last executed command:
```
Comment (kbdharun) [inline pages/common/echo.md:26]:
```suggestion
- Print the exit status of the last executed command (Note: In Windows Command Prompt, the equivalent command is `echo %errorlevel%`):
```

**Review by kbdharun — APPROVED** [review]

**Comment by shreenidhi11** [comment]
Hi All, 
Thanks for considering this contribution. I'm really looking forward to making more contributions in the future :)

**Pull request merged**